src/generators/server-side-rendering/index.ts

```diff
@@ -178,7 +178,12 @@ export default function ssr(
 		function __escape ( html ) {
 			return String( html ).replace( /["'&<>]/g, match => escaped[ match ] );
 		}
-	`.replace(/(\\)?@(\w*)/g, (match: string, escaped: string, name: string) => escaped ? match.slice(1) : generator.alias(name));
+	`.replace(/(\\)?([@#])(\w*)/g, (match: string, escaped: string, sigil: string, name: string) => {
```

Comment:
There are no `#`-sigiled identifiers in SSR code, because that means 'create a unique name inside this block', and there are no 'blocks' to speak of (because everything is inline). But because it uses the same `escape` helper, `#` characters are escaped anyway, and therefore need to be unescaped. A bit circuitous, admittedly